hutch_python/load_conf.py

```diff
@@ -62,16 +62,18 @@ def get_plugins(conf):
     """
     plugins = defaultdict(list)
 
-    for plugin_name in conf.keys():
+    for plugin_name, info in conf.items():
         try:
             module = import_module('hutch_python.plugins.' + plugin_name)
         except ImportError:
             module = None
             err = 'Plugin {} is not available, skipping'
             logger.warning(err.format(plugin_name))
```

Comment:
On an unrelated note this should probably be bumped to error.

Reply:
We should talk about what we want each of the log levels to mean, since they pop up in different colors during the load. Maybe something like:
```
Error (red): Malformed configurations, user-caused exceptions, etc.
Warning (yellow): Device failed to load
```
Eh, maybe that's not quite right either. The point I'm trying to make is that it's not obvious to me which of these elevated log statements should be red vs yellow.